### DISCUSSION TITLE
Data Center Notes

### DISCUSSION THREAD

**Discussion by bclanton40** [netbox-community/netbox#11130] [General]
Has anyone customized Netbox to include a global field for general notes that is applicable to the site.   For instance, we have a team of technicians that frequently visit the data center and we are required to itemize the work we have done on that particular day.   We want the tech to be able to enter his notes and be able to refer back to them by looking up the date.

Is that possible with Netbox?

**Comment by xkilian** [comment]
It should be handled by an external system. This external system could call netbox to get the correct site name and locations.
Could also be a custom web app that submits its data to a logging system or to an erp or a dcim...

**Comment by xkilian** [comment]
Netbox could have a custom url that beings you to the interface with a prepopulated filter to show the actions for a given site or location. This is already possible.

**Comment by jeremystretch** [comment]
NetBox supports journaling for individual objects, though I can't tell from your description if that would be a suitable solution.

**Comment by eronlloyd** [comment]
A plugin could support this. I'm considering developing a ticketing plugin because we're beginning to use Netbox more like an operational support system (OSS), anyhow. The journaling option might work in a pinch, too.
